xterm-color.el

```diff
@@ -831,24 +920,26 @@ effect when called from a buffer that does not have a cache."
 (defun xterm-color-test ()
   "Create/display and render a new buffer that contains ANSI control sequences."
```

Comment:
The diff in this function is noisy due to an indentation change. Telling Github to hide whitespace, it looks like this:

<table><tr><td><img width=400px src="https://user-images.githubusercontent.com/52205/79035462-fb4d3380-7b8c-11ea-896a-cbf32830eecb.png" alt="image" /></td></tr></table>